src/views/GRTSystemIntakeReview/index.tsx

```diff
@@ -34,210 +25,29 @@ export const GRTSystemIntakeReview = ({
       dispatch(getSystemIntake(await match.params.systemId));
     };
     fetchSystemIntake();
-  }, [auth, dispatch, match.params.systemId]);
+  }, [dispatch, match.params.systemId]);
   const systemIntake = useSelector(
     (state: AppState) => state.systemIntake.systemIntake
   );
-  const fundingDefinition = () => {
-    const isFunded = convertBoolToYesNo(!!systemIntake.fundingSource);
-    if (systemIntake.fundingSource) {
-      return `${isFunded}, ${systemIntake.fundingSource}`;
-    }
-    return isFunded;
-  };
-  const issoDefinition = () => {
-    const hasIsso = convertBoolToYesNo(!!systemIntake.isso);
-    if (systemIntake.isso) {
-      return `${hasIsso}, ${systemIntake.isso}`;
-    }
-    return hasIsso;
-  };
-  const anyCollaborators = () => {
-    if (
-      systemIntake.trbCollaborator ||
-      systemIntake.oitSecurityCollaborator ||
-      systemIntake.eaCollaborator
-    ) {
-      return true;
-    }
-    return false;
-  };
-
   return (
```

Comment:
This `return` is missing some things. It's missing the `Header` and the parent container isn't wrapped in the parent container to restrict content overflow. It should be wrapped by `className="grid-container"`.